stable_baselines3/common/utils.py

```diff
@@ -78,6 +78,35 @@ def update_learning_rate(optimizer: th.optim.Optimizer, learning_rate: float) ->
         param_group["lr"] = learning_rate
 
 
+class FloatConverterSchedule:
```

Comment:
This class actually does more no? it enforces that we have a schedule object that can be pickled, no?
Maybe rename it to `FloatSchedule` and update the docstring to explain its utility (in addition to casting to float)

Reply:
same comment as above